.github/workflows/kics.yml

```diff
@@ -1,28 +1,8 @@
-#################################################################################
```

Comment:
Copyright header needs to stay. You can read about this in the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers).
Also, if the file is changed, check the years are up to date